masterlist.yaml

```diff
@@ -808,47 +906,112 @@ plugins:
+        name: 'Bring Out Your Dead on Bethesda.net'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
     global_priority: -90
   - name: 'Darkwater Crossing.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/326/'
+        name: 'Darkwater Crossing on Nexus Mods'
+      - link: 'https://afkmods.iguanadons.net/index.php?/files/file/1895-darkwater-crossing/'
+        name: 'Darkwater Crossing on Afk Mods'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/2949694'
+        name: 'Darkwater Crossing on Bethesda.net'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
       - 'Immersive Citizens - AI Overhaul.esp'
   - name: 'Helarchen Creek.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/4043'
+        name: 'Helarchen Creek on Nexus Mods'
+      - link: 'https://afkmods.iguanadons.net/index.php?/files/file/1927-helarchen-creek/'
+        name: 'Helarchen Creek on Afk Mods'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/3090713'
+        name: 'Helarchen Creek on Bethesda.net'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'Ivarstead.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/349'
+        name: 'Ivarstead on Nexus Mods'
+      - link: 'https://afkmods.iguanadons.net/index.php?/files/file/1583-ivarstead/'
+        name: 'Ivarstead on Afk Mods'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/2958352'
+        name: 'Ivarstead on Bethesda.net'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'Karthwasten.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/350/'
+        name: 'Karthwasten on Nexus Mods'
+      - link: 'https://afkmods.iguanadons.net/index.php?/files/file/1768-karthwasten/'
+        name: 'Karthwasten on Afk Mods'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/2960640'
+        name: 'Karthwasten on Bethesda.net'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'Kynesgrove.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/351/'
+        name: 'Kynesgrove on Nexus Mods'
+      - link: 'https://afkmods.iguanadons.net/index.php?/files/file/1576-kynesgrove/'
+        name: 'Kynesgrove on Afk Mods'
+      - link: 'https://bethesda.net/en/mods/skyrim/mod-detail/2948900'
+        name: 'Kynesgrove on Bethesda.net'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'Shor''s Stone.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/354/'
+        name: 'Shor''s Stone on Nexus Mods'
+      - link: 'https://afkmods.iguanadons.net/index.php?/files/file/1908-shors-stone/'
+        name: 'Shor''s Stone on Afk Mods'
+      - link: 'http://www.nexusmods.com/skyrimspecialedition/mods/354/'
```

Comment:
Looks like a copy/paste error, it's not a bethesda.net URL.

Reply:
Yeah, it has been corrected.